metadata/src/main/java/org/apache/kafka/image/ClientQuotaImage.java

```diff
@@ -45,15 +45,15 @@ public final class ClientQuotaImage {
     private final Map<String, Double> quotas;
 
     public ClientQuotaImage(Map<String, Double> quotas) {
-        this.quotas = quotas;
+        this.quotas = Collections.unmodifiableMap(quotas);
     }
 
     Map<String, Double> quotas() {
         return quotas;
     }
 
     public Map<String, Double> quotaMap() {
-        return Collections.unmodifiableMap(quotas);
+        return this.quotas;
```

Comment:
we have a duplicate method now for accessing the quotas map, can we get rid of one?

Reply:
nice find